src/components/Sentence/SentenceCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import ClaimCardHeader from "../Claim/ClaimCardHeader";
+import ClaimReviewCardContent from "../ClaimReview/ReviewCardCommentContent";
+import TagsList from "../ClaimReview/TagsList";
+import ClaimReviewCardActions from "../ClaimReview/ReviewCardActions";
+import { ContentModelEnum } from "../../types/enums";
+import styled from "styled-components";
+import { Comment } from "antd";
+import CardBase from "../CardBase";
+import { useAtom } from "jotai";
+import { currentNameSpace } from "../../atoms/namespace";
+import { NameSpaceEnum } from "../../types/Namespace";
+
+const StyledComment = styled(Comment)`
+    .ant-comment-actions > li {
+        width: 100%;
+    }
+`;
+
+const SentenceCard = ({ sentence }) => {
+    const { content, claim, personality } = sentence;
+    const contentModel = sentence?.claim[0].contentModel;
+    const [nameSpace] = useAtom(currentNameSpace);
+    const isImage = contentModel === ContentModelEnum.Image;
+    const isDebate = contentModel === ContentModelEnum.Debate;
+
+    const buildReviewHref = (
+        nameSpace,
+        sentence,
+        personality,
+        isImage,
+        isDebate
+    ) => {
+        const baseNamespace =
+            nameSpace !== NameSpaceEnum.Main ? `/${nameSpace}` : "";
+        let pathSegment = "";
+
+        if (isDebate) {
```

Comment:
suggestion: I have the feeling this could be moved to a util library, what do you think?